### PR TITLE
Move metadata ingestion to usecase package

### PR DESCRIPTION
**Pull request by Pawel-608**

### CODE REVIEW

File: live-ingester/src/main/java/org/alfresco/hxi_connector/live_ingester/domain/usecase/metadata/IngestMetadataCommandHandler.java [hunk 1]
```diff
@@ -24,30 +24,27 @@
...
-import static org.alfresco.hxi_connector.live_ingester.domain.event.PredefinedNodeProperty.ASPECTS_NAMES;
-import static org.alfresco.hxi_connector.live_ingester.domain.event.PredefinedNodeProperty.CREATED_AT;
-import static org.alfresco.hxi_connector.live_ingester.domain.event.PredefinedNodeProperty.CREATED_BY_USER_WITH_ID;
-import static org.alfresco.hxi_connector.live_ingester.domain.event.PredefinedNodeProperty.IS_FILE;
-import static org.alfresco.hxi_connector.live_ingester.domain.event.PredefinedNodeProperty.IS_FOLDER;
-import static org.alfresco.hxi_connector.live_ingester.domain.event.PredefinedNodeProperty.MODIFIED_BY_USER_WITH_ID;
-import static org.alfresco.hxi_connector.live_ingester.domain.event.PredefinedNodeProperty.NAME;
-import static org.alfresco.hxi_connector.live_ingester.domain.event.PredefinedNodeProperty.PRIMARY_ASSOC_Q_NAME;
-import static org.alfresco.hxi_connector.live_ingester.domain.event.PredefinedNodeProperty.TYPE;
+import static org.alfresco.hxi_connector.live_ingester.domain.usecase.metadata.model.PredefinedNodeProperty.*;
```
Comment (krdabrowski) [inline live-ingester/src/main/java/org/alfresco/hxi_connector/live_ingester/domain/usecase/metadata/IngestMetadataCommandHandler.java:29]: Avoid wildcard imports

File: live-ingester/src/test/java/org/alfresco/hxi_connector/live_ingester/domain/usecase/metadata/IngestMetadataCommandHandlerTest.java [hunk 2]
```diff
@@ -24,36 +24,32 @@
...
-import static org.alfresco.hxi_connector.live_ingester.domain.event.PredefinedNodeProperty.ASPECTS_NAMES;
-import static org.alfresco.hxi_connector.live_ingester.domain.event.PredefinedNodeProperty.CREATED_AT;
-import static org.alfresco.hxi_connector.live_ingester.domain.event.PredefinedNodeProperty.CREATED_BY_USER_WITH_ID;
-import static org.alfresco.hxi_connector.live_ingester.domain.event.PredefinedNodeProperty.IS_FILE;
-import static org.alfresco.hxi_connector.live_ingester.domain.event.PredefinedNodeProperty.IS_FOLDER;
-import static org.alfresco.hxi_connector.live_ingester.domain.event.PredefinedNodeProperty.MODIFIED_BY_USER_WITH_ID;
-import static org.alfresco.hxi_connector.live_ingester.domain.event.PredefinedNodeProperty.NAME;
-import static org.alfresco.hxi_connector.live_ingester.domain.event.PredefinedNodeProperty.PRIMARY_ASSOC_Q_NAME;
-import static org.alfresco.hxi_connector.live_ingester.domain.event.PredefinedNodeProperty.TYPE;
+import static org.alfresco.hxi_connector.live_ingester.domain.usecase.metadata.model.PredefinedNodeProperty.*;
```
Comment (krdabrowski) [inline live-ingester/src/test/java/org/alfresco/hxi_connector/live_ingester/domain/usecase/metadata/IngestMetadataCommandHandlerTest.java:32]: Avoid wildcard imports

**Review by krdabrowski — APPROVED** [review]
LGTM, only fix your IDE config to avoid wildcard imports

File: live-ingester/src/main/java/org/alfresco/hxi_connector/live_ingester/domain/usecase/metadata/IngestMetadataCommandHandler.java (hunk 1 above)
Comment (Pawel-608) [reply live-ingester/src/main/java/org/alfresco/hxi_connector/live_ingester/domain/usecase/metadata/IngestMetadataCommandHandler.java:29]: meh, pmd should fail on this

File: live-ingester/src/test/java/org/alfresco/hxi_connector/live_ingester/domain/usecase/metadata/IngestMetadataCommandHandlerTest.java (hunk 2 above)
Comment (Pawel-608) [reply live-ingester/src/test/java/org/alfresco/hxi_connector/live_ingester/domain/usecase/metadata/IngestMetadataCommandHandlerTest.java]: fixed

File: live-ingester/src/main/java/org/alfresco/hxi_connector/live_ingester/domain/usecase/metadata/IngestMetadataCommandHandler.java (hunk 1 above)
Comment (Pawel-608) [reply live-ingester/src/main/java/org/alfresco/hxi_connector/live_ingester/domain/usecase/metadata/IngestMetadataCommandHandler.java]: fixed

**Pull request merged**